android/app/build.gradle

```diff
@@ -103,8 +103,8 @@ android {
         applicationId "com.zooniversemobile"
         minSdkVersion 16
         targetSdkVersion 27
-        versionCode 17
-        versionName "2.1.0"
+        versionCode 23
+        versionName "2.2.7"
```

Comment:
Why such the increment? Was this all in getting Fastlane setup?

Reply:
Yeah, are version numbering is fairly arbitrary already. And I had to push a lot of builds up to the app store/play store consoles (production wasn't updated) for testing. So I figured I would just leave the versioning at what it ended up at at the end of testing

Reply:
That's fine. I recall the stores being picks about uping the version whenever a push is made.